gradle/verification-metadata.xml

```diff
@@ -1051,6 +1051,11 @@
             <pgp value="77993d3295698c46912e83984b2277c6358670ee"/>
          </artifact>
       </component>
+      <component group="org.assertj" name="assertj-core" version="3.20.2">
```

Comment:
Is it possible to consolidate the assertj versions we use? I think it would be great to only use 3.20 (or whathever the newest version is) instead of three different versions.

Reply:
It's `apiguardian-api.jar`. I guess it must have been dropped from JUnit?

Reply:
I removed one that is not used, but this problem would be better solved by enabling `--write-verification-metadata` to remove unused entries. I believe @ljacomet will implement that at some point.